test/extensions/filters/http/aws_lambda/aws_lambda_transformer_test.cc

```diff
@@ -60,19 +60,26 @@ class AWSLambdaTransformerTest : public testing::Test,
 protected:
   void SetUp() override { setupRoute(); }
 
-  const std::string TRANSFORMATION_YAML =
+  const std::string REQUEST_TRANSFORMATION_YAML =
   R"EOF(
-  name: io.solo.transformer.api_gateway_test_transformer
+  name: io.solo.requesttransformer.api_gateway_test_transformer
   typed_config:
-    "@type": type.googleapis.com/envoy.test.extensions.transformation.ApiGatewayTestTransformer
```

Comment:
this is a break in backwards compatibility. even if deprecated, we should make sure the old format works for at least a release before removing so we have a zero downtime upgrade path. otherwise during upgrade if control plane upgrades and sends new APIs before dataplane updates then we can have dataplane downtime

Reply:
if this does in fact still work, we should have the old test remain and just add new ones. and add appropriate deprecation warnings for removal in next Gloo Edge/Gloo platform release (we don't support jump upgrades)

Reply:
The types of the actual transformers are not changing. The wrapAsApiGateway transformer (on the request path) is in EE and will be updated to be a `RequestTransformer` only. The unwrapAsApiGateway transformer (on the response path) is now in OSS and is a `ResponseTransformer` only.

The reason I needed to distinguish here is because the fake transformer used for testing was registered only once (as a `Transformer`). So to use a distinct `RequestTransformer` for request path and `ResponseTransformer` for response path, I had to make this distinction in the tests.